libs/brig-types/src/Brig/Types/Team/LegalHold.hs

```diff
@@ -241,6 +241,15 @@ instance ToJSON LegalHoldServiceConfirm where
         #  "refresh_token" .= refreshToken
         # []
 
+instance FromJSON LegalHoldServiceConfirm where
```

Comment:
Remind me to show you where we do JSON round-trip tests and we can add this there :)